### PR TITLE
[Github] add spirv backend team

### PR DESCRIPTION
**Pull request by Keenuts**
This PR adds the paths the SPIR-V backend folks maintains. (Another distince SPIR-V team exists, but only for the MLIR part (mlir-spirv).

@tstellar : the team doesn't exist, I saw a message on the discourse asking people to create a PR and add you. Hope this is the correct process 😊

### CODE REVIEW

**Comment by tstellar** [comment]
Can you move these paths into [new-prs-labeler.yml](https://github.com/llvm/llvm-project/blob/main/.github/new-prs-labeler.yml).  We are no longer using the CODEOWNERS file for notifications.

**Comment by Keenuts** [comment]
> Can you move these paths into [new-prs-labeler.yml](https://github.com/llvm/llvm-project/blob/main/.github/new-prs-labeler.yml). We are no longer using the CODEOWNERS file for notifications.

Right, matched the label name. Do you still need to create a new team first or shall we just merge this once approved?

**Comment by tstellar** [comment]
Team created: https://github.com/orgs/llvm/teams/pr-subscribers-backend-spir-v/

**Review by tstellar — APPROVED** [review]

**Review by michalpaszkowski — APPROVED** [review]

**Pull request merged**
